.github/workflows/test.yaml

```diff
@@ -186,6 +186,7 @@ jobs:
           - almalinux-latest-amd64-openssl-1.1.x
           - almalinux-latest-arm64-openssl-1.1.x
           - alpine-3.16-amd64-openssl-1.1.x
+          - alpine-3.17-amd64-openssl-1.1.x
```

Comment:
Unrelated: I think it would be a good idea to add the compat stuff (or manualy installation etc) in the name here so one knows where this openssl version comes from for each - makes it easier to understand that is happening. Follow up PR of course.